framework/src/vectorpostprocessors/StatisticsVectorPostprocessor.C

```diff
@@ -0,0 +1,144 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "StatisticsVectorPostprocessor.h"
+
+// MOOSE includes
+#include "MooseVariableField.h"
+#include "ThreadedElementLoopBase.h"
+#include "ThreadedNodeLoop.h"
+
+#include "libmesh/quadrature.h"
+
```

Comment:
`#include <numeric>`

Reply:
Because you need it: https://civet.inl.gov/job/174638/

Reply:
That's not what caused that failure.  The failure was an actual copy-and-paste bug that you missed: https://github.com/idaholab/moose/pull/11216/files#diff-5e0bb82e85bf3311c34cfcf6d004e3e9R13

Reply:
Hmm, Are you sure? Doing a "grep" shows that we only have three includes of "numeric" in the whole framework. Two of them in vector postprocessors and one in `MooseUtils.h`. The unity build will work because of the former, but we are definitely pushing the lucky side of things if we are relying on MooseUtils to pull it in through an indirect include.

Reply:
I `#included` it to be sure